### PR TITLE
docs: better documentation for `runtime`

### PR DESCRIPTION
**Pull request by ematipico**
## Changes

Closes https://github.com/withastro/astro/issues/8060

## Testing

N/A

## Docs

/cc @withastro/maintainers-docs for feedback!

### CODE REVIEW

**Comment by ematipico** [comment]
cc https://github.com/alexanderniebuhr should we add something about how to type the runtime?

**Comment by alexanderniebuhr** [comment]
> cc @alexanderniebuhr should we add something about how to type the runtime?

Sure thing, let me double check how typing would be best with the current types & will open another PR. Should not block this one.

File: packages/integrations/cloudflare/README.md [hunk 1]
```diff
@@ -75,10 +75,23 @@ It's then possible to update the preview script in your `package.json` to `"prev
 
 You can access all the Cloudflare bindings and environment variables from Astro components and API routes through `Astro.locals`.
 
-```js
+If you're inside an `.astro` file, you access the runtime using the `Astro.locals` global:
+
+```astro
 const env = Astro.locals.runtime.env;
```
Comment (alexanderniebuhr) [inline packages/integrations/cloudflare/README.md:81]: What do you think about this change? In the sentence above you talked about `runtime`, so I think it makes sense to update the example, not sure if we should show that `env` is inside it.
```suggestion
const runtime = Astro.locals.runtime;
```

**Review by alexanderniebuhr — COMMENTED** [review]
This makes sense. 🎉

**Comment by ematipico** [comment]
@alexanderniebuhr I committed a change where the types are now exported and improved the docs showing how users can type the runtime. Please have a look!

**Comment by alexanderniebuhr** [comment]
I like it!

**Review by sarah11918 — APPROVED** [review]
Docs approves! :partying_face:

**Pull request merged**